JASP-Engine/JASP/R/commonMachineLearningClustering.R

```diff
@@ -38,11 +48,15 @@
     }
   }
 
-  if(length(predictors[predictors != ""]) > 0L)
-    .hasErrors(dataset, type = c('infinity', 'observations'), custom=checkClusters, all.target = predictors,
-               observations.amount = "< 2", exitAnalysisIfErrors = TRUE)
+  checkDataHierarchicalClustering <- function() {
+    if(type != "hierarchical")
+      return()
 
-  return()
+    if(nrow(dataset) >= 65536L)
+      return("R package error: The hclust clustering algorithm from the stats R package cannot handle data that has 65536 or more rows. Please try another algorithm, as we are working towards a solution for this problem.")
```

Comment:
We are working on a solution for this problem. Please try another algorithm in the meantime.